packages/workspace/src/workspace/codebase/make-directory/index.ts

```diff
@@ -3,6 +3,26 @@ import { basename, dirname, join } from 'path';
 import type { Codebase } from '..';
 import type { FileContainer } from '../file';
 
+// TODO: the file has to be the of the same codebase? Do we need to pass codebase? Can we get it from file?
```

Comment:
You're right. This is the function that creates a new folder and moves the function to an `index` file in Slimfast. It's just very much a workspace sort of function, so it exists here to add the utility.